pepper-apis/src/main/java/org/broadinstitute/ddp/db/TransactionWrapper.java

```diff
@@ -158,24 +146,39 @@ public DB getDb() {
         }
     }
 
-    public static synchronized void init(DbConfiguration...dbConfigs) {
+    public static synchronized void init(String defaultTimeZoneName, DbConfiguration...dbConfigs) {
+        if (dbConfigs.length == 0) {
+            throw new IllegalArgumentException("must have at least one dbConfig");
+        }
+
         for (DbConfiguration dbConfig : dbConfigs) {
             if (gTxnWrapper.containsKey(dbConfig.getDb())) {
                 TransactionWrapper transactionWrapper = gTxnWrapper.get(dbConfig.getDb());
                 int maxConnections = dbConfig.getMaxConnections();
                 String dbUrl = dbConfig.getDbUrl();
                 if (transactionWrapper.maxConnections != maxConnections || !transactionWrapper.dbUrl.equals(dbUrl)) {
                     throw new RuntimeException("init() has already been called with "
-                                               + transactionWrapper.maxConnections + " and "
-                                               + transactionWrapper.dbUrl + "; " +  "you cannot re-initialize "
-                                               + "it with different params " + maxConnections
-                                               + " and " + dbUrl);
+                            + transactionWrapper.maxConnections + " and "
+                            + transactionWrapper.dbUrl + "; " +  "you cannot re-initialize "
+                            + "it with different params " + maxConnections
+                            + " and " + dbUrl);
                 } else {
                     LOG.warn("TransactionWrapper has already been initialized.");
                 }
             }
-            gTxnWrapper.put(dbConfig.getDb(), new TransactionWrapper(dbConfig.getMaxConnections(), dbConfig.getDbUrl()));
+            gTxnWrapper.put(dbConfig.getDb(), new TransactionWrapper(dbConfig.getMaxConnections(), dbConfig.getDbUrl(), dbConfig.getDb()));
+        }
+        // We are setting the VMs default timezone expectation that it will be same on server
+        // or at least specified in DB connection URL
+        // if timezones don't match, times and dates might be miscalculated/improperly converted by JDBC connector
+        // https://bugs.mysql.com/bug.php?id=91112
+        LOG.info("Setting default time zone to: {}", defaultTimeZoneName);
+        TimeZone defaultTimeZone = TimeZone.getTimeZone(defaultTimeZoneName);
+        if (defaultTimeZone == null) {
+            throw new RuntimeException(String.format("Could not find a Time zone with name: %s", defaultTimeZoneName));
         }
+        TimeZone.setDefault(defaultTimeZone);
```

Comment:
Can we set the default timezone when we bootstrap the server instead of within TransactionWrapper? How about somewhere in `DataDonationPlatform.main()`?